setup.py

```diff
@@ -118,6 +118,7 @@
     "flax>=0.4.1,<=0.6.9",
     "ftfy",
     "fugashi>=1.0",
+    "g2p-en>=2.1.0",
```

Comment:
"Just" the g2p_en.G2p().__call__ method, and it's only called within `_tokenize`. Its fairly nested and I pulled it out into a notebook to illustrate what we'd be re-implementing [here](https://colab.research.google.com/drive/1u4utpaYgthPZ3rXmdeucTYB-8r81SUPe). It's a couple hundred lines and  notably still requires packages like nltk and doesn't include the long-ish data from the `homograph.en` file used to create the homograph2features dictionary. The amount of work / code to reimplement it would also depend on how stringently we choose to match the outputs to the original.

Imo it's best to simply keep `g2p_en`, but it's hard for me to evaluate the tradeoff without knowing more about how the maintainers view deps like this.